tensorflow/contrib/tensorrt/convert/convert_nodes.cc

```diff
@@ -95,6 +95,9 @@ inline tensorflow::Status ConvertDType(tensorflow::DataType tf_dtype,
     case tensorflow::DataType::DT_INT8:
       *trt_dtype = nvinfer1::DataType::kINT8;
       break;
+    case tensorflow::DataType::DT_QINT8:
```

Comment:
No they're not the same:

- DT_INT8 is not quantize type, it doesn't require min/max range information for computation (similar to DT_INT32, DT_FLOAT32, etc)
- DT_QINT8 is quantize type and we do need range information for any computation. I think this is equivalent to TRT's INT8, since TRT's INT8 needs either calibration to populate the ranges or use setDynamicRange(). This is why I think converting DT_INT8 to TRT's INT8 may be wrong.